app/src/org/commcare/activities/AppManagerActivity.java

```diff
@@ -90,10 +92,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
 
     protected void showAboutCommCareDialog() {
-        CommCareAlertDialog dialog = DialogCreationHelpers.buildAboutCommCareDialog(this, false);
-        dialog.makeCancelable();
-        dialog.setOnDismissListener(dialog1 -> handleDeveloperModeClicks());
-        showAlertDialog(dialog);
+        Log.d("Shivamm", "Another lint issue");
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/b67bf14163879f9fb09208858b78318e65c45843/app/src/org/commcare/activities/AppManagerActivity.java#L95"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>